x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -598,6 +625,52 @@ export class AlertsClient {
     }
   }
 
+  public async find<Params extends AlertTypeParams = never>({
+    query,
+    aggs,
+    _source,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    track_total_hits,
```

Comment:
Nit: Probably should prefer `track_total_hits: trackTotalHits` here, instead of the eslint exception?